unit/modules/avgMol/water.cpp

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#include "unit/data/systemTest.h"
+#include <gtest/gtest.h>
+#include <vector>
+
+namespace UnitTest
+{
+class AvgMolModuleTest : public ::testing::Test
+{
+    public:
+    AvgMolModuleTest(){};
+
+    protected:
+    DissolveSystemTest systemTest;
+
+    void SetUp() override
+    {
+        EXPECT_NO_THROW(systemTest.setUp("../unit/modules/avgMol/water.txt"));
```

Comment:
```suggestion
        ASSERT_NO_THROW(systemTest.setUp("../unit/modules/avgMol/water.txt"));
```
Maybe it is better to `assert` here as it doesn't make sense for the test to continue if dissolve isn't set up?